packages/core/src/dropdown/Dropdown.css

```diff
@@ -49,52 +67,102 @@
+.saltDropdown[aria-readonly="true"]:focus {
+  --dropdown-background: var(--dropdown-background-readonly);
+  --dropdown-borderColor: var(--salt-editable-borderColor-readonly);
 }
 
-.saltDropdown-error .saltDropdown:focus-visible {
-  outline-color: var(--salt-status-error-borderColor);
+.saltDropdown-error,
+.saltDropdown-error:hover,
+.saltDropdown-error:focus {
+  --dropdown-background: var(--salt-status-error-background);
   --dropdown-borderColor: var(--salt-status-error-borderColor);
+  --dropdown-background-active: var(--salt-status-error-background);
+  --dropdown-background-hover: var(--salt-status-error-background);
+  --dropdown-background-readonly: var(--salt-status-error-background);
+  outline-color: var(--salt-status-error-borderColor);
 }
 
-.saltDropdown-warning {
+.saltDropdown-warning,
+.saltDropdown-warning:hover,
+.saltDropdown-warning:focus {
   --dropdown-background: var(--salt-status-warning-background);
   --dropdown-borderColor: var(--salt-status-warning-borderColor);
-}
-
-.saltDropdown-warning .saltDropdown:focus-visible {
+  --dropdown-background-active: var(--salt-status-warning-background);
+  --dropdown-background-hover: var(--salt-status-warning-background);
+  --dropdown-background-readonly: var(--salt-status-warning-background);
   outline-color: var(--salt-status-warning-borderColor);
-  --dropdown-borderColor: var(--salt-status-warning-borderColor);
 }
 
-.saltDropdown-success {
+.saltDropdown-success,
+.saltDropdown-success:hover,
+.saltDropdown-success:focus {
   --dropdown-background: var(--salt-status-success-background);
   --dropdown-borderColor: var(--salt-status-success-borderColor);
+  --dropdown-background-active: var(--salt-status-success-background);
+  --dropdown-background-hover: var(--salt-status-success-background);
+  --dropdown-background-readonly: var(--salt-status-success-background);
+  outline-color: var(--salt-status-success-borderColor);
 }
 
-.saltDropdown-success .saltDropdown:focus-visible {
-  outline-color: var(--salt-status-success-borderColor);
-  --dropdown-borderColor: var(--salt-status-success-borderColor);
+.saltDropdown-activationIndicator {
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  position: absolute;
+  border-bottom: var(--dropdown-borderWidth) var(--dropdown-borderStyle) var(--dropdown-borderColor);
+}
+
+.saltDropdown-bordered {
+  --dropdown-border: var(--salt-size-border) var(--salt-container-borderStyle) var(--dropdown-borderColor);
+  --dropdown-borderWidth: 0;
+}
+
+/* Style applied if focused or active when `bordered={true}` */
+.saltDropdown-bordered.saltDropdown:focus,
+.saltDropdown-bordered:active {
+  --dropdown-borderWidth: var(--salt-editable-borderWidth-active);
+}
+
+/* Styling when focused if `disabled={true}` or `readOnly={true}` when `bordered={true}` */
+.saltDropdown-bordered[aria-readonly="true"],
+.saltDropdown-bordered[aria-readonly="true"]:hover,
+.saltDropdown[aria-readonly="true"]:focus,
+.saltDropdown-bordered.saltDropdown-disabled,
```

Comment:
Why does this one use -disabled vs :disabled

Reply:
changes to :disabled